frontend/client/components/ContributionModal/index.tsx

```diff
@@ -127,17 +128,14 @@ class ContributionModal extends React.Component<Props, State> {
           message="This contribution will not be attributed"
           description={
             <>
-              Your contribution will show up without attribution. Even if you're logged
-              in, the contribution will not appear anywhere on your account after you
-              close this modal.
+              Your contribution will show up without attribution.
               <br /> <br />
               ZF Grants is unable to offer refunds for non-attributed contributions. If
```

Comment:
This is no longer the case, since refunds will be issued as long as you have a refund address set at the account level.

Reply:
This message only shows for anonymous/unauthenticated users now.

Reply:
Ah, my mistake.